src/makerom/rom_header.s

```diff
@@ -23,4 +28,12 @@
 #elif OOT_REGION == REGION_EU
 /* 0x3E */ REGION(PAL)
 #endif
+#if OOT_VERSION == NTSC_1_0 || OOT_VERSION == PAL_1_0
+/* 0x3F */ GAME_REVISION(0)
+#elif OOT_VERSION == NTSC_1_1 || OOT_VERSION == PAL_1_1
+/* 0x3F */ GAME_REVISION(1)
+#elif OOT_VERSION == NTSC_1_2
+/* 0x3F */ GAME_REVISION(2)
```

Comment:
I never knew game revision was properly set for the n64 versions.
Maybe we should consider our n64 version names to just be revision numbers. as in `NTSC_0` `NTSC_1` `NTSC_2`.
The `1_X` convention was always a little unfortunate as an enum anyway. And comments can still indicate the "colloquial" 1.0, 1.1, and 1.2

Reply:
I think `NTSC_1_0` at least suggests "NTSC 1.0" whereas `NTSC_0` is more mysterious, and I don't really want to comment every usage of it

Reply:
I'm suggesting that internally we call it revision 0 (NTSC_0) instead of 1.0, which seems more proper, as opposed to the made up versioning that speedrunners created 

The comments could be in easy to see places, like the make file and enum, not every usage

Reply:
I'd say "revision 0" is also a made up naming scheme, but even if we did know the official names, at this point I would still prefer to call it 1.0. The point of names is communication after all, and the name 1.0 has the huge advantage of being well-established everywhere (not just in speedrunning but also modding, randomizer, etc). Calling it "revision 0" would not add any clarity and would be needlessly confusing

Reply:
I mean, the numbers are right here in the ROM header. They aren't made up. It's 0 for "1.0", 1 for "1.1" and 2 for "1.2".
So ntsc-0 ntsc-1 and ntsc-2 would make the most sense to me. And the enum looks better in the code this way

Reply:
Yeah maybe those names are more "proper" in some sense, but ultimately I think it's a worse name, because the point of a name is not to be proper, it's to communicate some idea to the reader

Reply:
I agree with fig here

Reply:
it wont hold this PR up at all, but I think its something to consider more and talk about separately soon

Reply:
This kinda reminds me to how floats are encoded in binary.
The leading `1.` part of the binary notation is implicit, meaning it is not actually stored on the binary representation to avoid wasting an extra bit for something that will be always set (except on subnormal floats)
So maybe this is a somewhat similar case? The `1.` part is implicit since this is an actual release and they never expected to make an OoT 2.0 or whatever.

Either way, I don't know if this mumbling actually contributes anything significant to the discussion. Feel free to ignore it lol